bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1428,6 +1428,14 @@ lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_1x1_mexicocityMEX_hist_2000_78pfts_c24
 <fsurdat hgrid="1x1_urbanc_alpha"    sim_year="2000" >
 lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_1x1_urbanc_alpha_hist_2000_78pfts_c240221.nc</fsurdat>
 
+<!-- for 1979 simulations (SSP2 label is irrelevant in these files) -->
+<fsurdat hgrid="ne0np4.ARCTICGRIS.ne30x8" sim_year="1979">
+lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_ne0np4.ARCTICGRIS.ne30x8_SSP2-4.5_1979_78pfts_c240425.nc</fsurdat>
+<fsurdat hgrid="ne0np4.ARCTIC.ne30x4"     sim_year="1979">
+lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_ne0np4.ARCTIC.ne30x4_SSP2-4.5_1979_78pfts_c240425.nc</fsurdat>
+<fsurdat hgrid="ne0np4CONUS.ne30x8"       sim_year="1979">
+lnd/clm2/surfdata_esmf/ctsm5.2.0/surfdata_ne0np4CONUS.ne30x8_SSP2-4.5_1979_78pfts_c240425.nc</fsurdat>
```

Comment:
From meeting with @ekluzek:
Rename these files to "hist" rather than "SSP..."

Longer term, look at the automatic naming convention regarding fsurdat files as part of #2498.